examples/mega65/CMakeLists.txt

```diff
@@ -4,3 +4,13 @@ install_example(viciv_test.prg)
 add_executable(plasma.prg plasma.cc)
 install_example(plasma.prg)
 
+add_executable(vertical_raster.prg vertical_raster.cc)
+install_example(vertical_raster.prg)
+
+add_executable(simple_dma.prg simple_dma.cc)
+install_example(simple_dma.prg)
+
+add_executable(dma_audio.prg dma_audio.c)
+install_example(dma_audio.prg)
+install(FILES drums.s8 TYPE BIN)
```

Comment:
Should `CMAKE_CURRENT_SOURCE_DIR` be added or is that implied? Seems to work correctly as is, though.

Reply:
This should be fine.